packages/web/components/complex/pool/create/step1-set-ratios.tsx

```diff
@@ -7,63 +7,68 @@ import { InputBox } from "../../../input";
 import { StepProps } from "./types";
 import { StepBase } from "./step-base";
 import { useWindowSize } from "../../../../hooks";
-import { Button } from "../../../buttons";
+import { BorderButton } from "../../../buttons";
+import { useTranslation } from "react-multi-lang";
 
 export const Step1SetRatios: FunctionComponent<StepProps> = observer(
   (props) => {
     const { createPoolConfig: config } = props;
     const { isMobile } = useWindowSize();
+    const t = useTranslation();
 
     return (
       <StepBase step={1} {...props}>
         <div className="flex flex-col gap-2.5">
           {config.assets.map(({ amountConfig, percentage }, index) => (
             <div
               key={amountConfig.sendCurrency.coinDenom}
-              className="h-24 md:h-auto flex px-7 md:p-2.5 items-center place-content-between border border-white-faint rounded-2xl"
+              className="flex items-center shrink-0 place-content-between h-24 md:h-auto px-7 md:p-2.5 border border-osmoverse-700 rounded-2xl"
             >
               <TokenSelect
                 selectedTokenDenom={amountConfig.sendCurrency.coinDenom}
-                tokens={config.sendableCurrencies}
+                tokens={config.remainingSelectableCurrencies.concat(
+                  amountConfig.sendCurrency
+                )}
                 onSelect={(coinDenom) => {
                   const currency = config.remainingSelectableCurrencies.find(
                     (currency) => currency.coinDenom === coinDenom
                   );
                   if (currency) {
                     amountConfig.setSendCurrency(currency);
+                  } else {
+                    console.error(
+                      "Unable to find currency selected in TokenSelect to be set in create pool config"
+                    );
```

Comment:
Do we need to make this an error object too?

Reply:
What do you mean by error object? It's basically to help debug.

Reply:
like this you made on `packages/stores/src/ui-config/errors.ts`
```ts
export class NegativeSwapFeeError extends Error {
  constructor(m: string) {
    super(m);
    // Set the prototype explicitly.
    Object.setPrototypeOf(this, NegativeSwapFeeError.prototype);
  }
}
```